conf/messages

```diff
@@ -22,4 +22,25 @@ startPage1 = You will be able to submit other income sources in the future.
 startPage2 = For self employment and property income, you need to use your chosen software.
 startPage.interest = interest
 startPage.dividends = dividends
-startPage.provideUpdates = Provide updates
\ No newline at end of file
+startPage.provideUpdates = Provide updates
+
+overview.agent.heading = Your Client''s Income Tax Return
+overview.caption = {0} to {1} Income Tax
+overview.agent.task1.heading = Tailor your client''s return
+overview.agent.task1.income = Add or remove income sources
+overview.agent.task2.heading = Provide updates
+overview.agent.task2.caption = complete the tailor your client''s return section before you can start making updates to your return.
+overview.agent.task3.heading = View estimates
+overview.agent.task3.caption = provide at least one update before you can view your client''s estimate.
```

Comment:
Could you capitalise the first letter of the captions so it's in line with the screenshots